program/src/oracle/oracle.c

```diff
@@ -510,6 +510,9 @@ static uint64_t upd_price( SolParameters *prm, SolAccountInfo *ka )
     fptr->conf_     = cptr->conf_;
     fptr->status_   = status;
     fptr->pub_slot_ = cptr->pub_slot_;
+    sol_log("Instruction: UpdatePrice");
```

Comment:
It might be nicer to log the `hdr->cmd_` enum value in the `dispatch` function, so that we log this for all instructions.

Reply:
YES!

Reply:
I like @tompntn 's suggestion. One way to get around the int issue is to make a table mapping the ints -> the expected log string.